cntp-appicontool/main.cpp

```diff
@@ -175,5 +179,28 @@ int main(int argc, char* argv[]) {
         rcFile.close();
     }
 
+    if (parser.isSet("output-png")) {
+        eoutput << "Creating PNG icon";
+        CombinedIcon pngIcon;
+        pngIcon.setBaseIcon(CombinedIcon::PlatformSpecificIcon);
+        pngIcon.setIconGradientColors(color1, color2);
+        pngIcon.setGenerateBlueprintIcon(parser.isSet("blueprint"));
+        pngIcon.setOverlayIcon(overlayIcon);
+        pngIcon.setOverlayIconMac(overlayIconMac);
+
+        QSize size(256, 256);
```

Comment:
We should generate all the right sizes and assets according to https://learn.microsoft.com/en-us/windows/apps/design/style/iconography/app-icon-construction. We also need to make sure that the sizing is correct